numpydoc/tests/test_docscrape.py

```diff
@@ -921,6 +926,20 @@ class BadSection:
 def test_empty_first_line():
     assert doc7['Summary'][0].startswith('Doc starts')
 
+doc8 = NumpyDocString("""
+
+        Parameters wit colon and no types:
+
+        Parameters
+        ----------
+
+        data :
+            some stuff, technically invalid
+        """)
+
+
+def test_trailing_colon():
+    assert doc8['Parameters'][0].name == 'data'
```

Comment:
PEP8 two newlines before new func (or after this one)
```suggestion
    assert doc8['Parameters'][0].name == 'data'

```